docs/getting-started.md

```diff
@@ -113,7 +111,7 @@ User.init({
 });
 ```
 
-Alternatively (legacy):
+Alternatively (legacy declaration, using `sequelize.define`):
```

Comment:
`sequelize.define` was marked legacy? when? I thought that was only for TypeScript. `init` requires two step where `define` is single step. It should be ok to use it on non-typescript projects.

Please remove `legacy` :)

Reply:
Oh, very well. Looks like it was a misunderstanding. Sorry about that. It wasn't me who added the `legacy` terminology for it, I just saw it being done and assumed it happened. In fact I did think it was a bit strange, so it is good from hear to you that it's not really legacy. Great! Will fix it later.

Reply:
Yea my bad, shouldn't have labeled legacy, I think you can call this `alternative (non es6)` or something along those lines.

Reply:
@sushantdhiman - Done! Let me know if you want more changes.